### ISSUE TITLE
Formatting Accesibilities Map

### ISSUE THREAD

**Issue by Lyrunis** [EECE3093C/team-project-python#52]
Current accessibility map is narrow and does not work with CSS class .mapstyle in findroutes.html. Manually override the width/height of map.
